airflow/www/static/js/dag/details/taskInstance/Logs/index.tsx

```diff
@@ -266,12 +267,14 @@ const Logs = ({
               <Text fontSize="sm">{warning}</Text>
             </Flex>
           )}
-          {!!parsedLogs && (
+          {parsedLogs ? (
             <LogBlock
               parsedLogs={parsedLogs}
               wrap={wrap}
               tryNumber={taskTryNumber}
             />
+          ) : (
+            <Spinner />
```

Comment:
Let's get the loading state from the query to determine when to show the spinner.
`const { data, isLoading } = useTaskLog()`

Reply:
Changes done. Since `isLoading` is always `false` in the above case because of the placeholder in the query. Hence, using `isFetching` for the spinner.

Reply:
Done